scripts/synthetix/facilicate_trade.js

```diff
@@ -0,0 +1,88 @@
+const { getExchange } = require("../utils/trading_strategy_helpers")(web3, artifacts)
+const { SynthetixJs } = require("synthetix-js")
+const { calculateBuyAndSellAmountsFromPrice, fetchTokenInfoFromExchange } = require("../utils/trading_strategy_helpers")(
+  web3,
+  artifacts
+)
+
+const sUSDAddress = {
+  mainnet: "0xAe38b81459d74A8C16eAa968c792207603D84480",
+  rinkeby: "0x1b642a124CDFa1E5835276A6ddAA6CFC4B35d52c",
+}
+
+const sETHAddress = {
+  mainnet: "0xD0DC005d31C2979CC0d38718e23c82D1A50004C0",
+  rinkeby: "0x0647b2C7a2a818276154b0fC79557F512B165bc1",
+}
+
+module.exports = async callback => {
+  try {
+    const snxjs = new SynthetixJs({ networkId: await web3.eth.net.getId() })
+    const exchange = await getExchange(web3)
+    const defaultAccount = (await web3.eth.getAccounts())[0]
+
+    // Both of these hardcoded tokens are assumed to have 18 decimal places.
+    // We "trust" that this will always be the case although it seems
+    // that synthetix reserves the authority to upgrade their token
+    // This could mean issuing a new one with a different number of decimals.
+    const sETHKey = await snxjs.sETH.currencyKey()
+    const sUSDKey = await snxjs.sUSD.currencyKey()
+
+    // Fetch token IDs and other relevant token information for order placement.
+    const referenceTokenId = (await exchange.tokenAddressToIdMap.call(sUSDAddress[process.env.NETWORK_NAME])).toNumber()
```

Comment:
nit: I'd reuse the value we get from web3.eth.net.getId() here to make sure we only rely on one way of getting the network ID.

Also for the variable name, would it make sense to call this susdTokenId or the like?

Reply:
Yes, this is a goo idea.. the change here will also go along with using the curated `tokenList.json`